server/parsec/backend/invite.py

```diff
@@ -289,55 +289,6 @@ def _on_status_changed(
             _on_status_changed,  # type: ignore[arg-type]
         )
 
-    @api
-    async def apiv2_invite_new(
-        self,
-        client_ctx: AuthenticatedClientContext,
-        req: authenticated_cmds.v2.invite_new.Req,
-    ) -> authenticated_cmds.v2.invite_new.Rep:
-        # `invite_new` command is similar between APIv2 and v4+ from the server
-        # point of view.
-        # (from client point of view, server may return `ok` response
-        # with some fields missing)
-        v2_unit = req.unit
-        unit: authenticated_cmds.latest.invite_new.UserOrDevice
-        if isinstance(v2_unit, authenticated_cmds.v2.invite_new.UserOrDeviceUser):
-            unit = authenticated_cmds.latest.invite_new.UserOrDeviceUser(
-                claimer_email=v2_unit.claimer_email,
-                send_email=v2_unit.send_email,
-            )
-        else:
-            assert isinstance(v2_unit, authenticated_cmds.v2.invite_new.UserOrDeviceDevice)
-            unit = authenticated_cmds.latest.invite_new.UserOrDeviceDevice(
-                send_email=v2_unit.send_email,
-            )
-        # TODO: proper api req/rep conversion
-        latest_req = authenticated_cmds.latest.invite_new.Req(unit)
-        return await self.api_invite_new(client_ctx, latest_req)  # type: ignore[return-value]
-
-    @api
-    async def apiv3_invite_new(
```

Comment:
APIv3 should stay !

Reply:
Yep! I removed it because otherwise there was an error when generating the schemas (see my comment in the PR description). I will re-add them and try to investigate more about that error.

Reply:
Ho yes you're right !

The `apiv3_xxx` method is required if the req/rep types are different in APIv3 than in APIv4, however some req/rep used to be different but this is no longer the case: since removing APIv2 remove some introduced_in fields.

This is because the heuristic on this field is really simple there: a schema with this field is never shared between APIs (i.e. if `introduced_in==2.1`, the schema in APIv3 and APIv4 could be shared but they are not)